hurumap_tz/settings.py

```diff
@@ -106,6 +119,37 @@
     ]
 }
 
+HURUMAP['showcase_stories'] = [
+    {
+        'title': 'Umbumbumbu wa Hesabu, Kiingereza unavyotishia mustakabali wa elimu Tanzania',
+        'author': 'Nuzulack Dausen',
+        'brief': 'Msingi dhaifu wa masomo ya Hisabati na Kiingereza unazidi kudhoofisha'\
+        ' elimu ya msingi nchini, jambo linalotishia mustakabali wa kuzalisha wataalamu bora watakaochochea maendeleo.',
+        'link': 'https://data.mwananchi.co.tz/umbumbumbu-wa-hesabu-kiingereza-unavyotishia-mustakabali-wa-elimu-tanzania/',
+        'img': STATIC_URL + 'img/showcase/uwezo.png'
+    },
+    {
+        'title': 'Hii ndiyo siri ya utofauti wa kipato kati ya wanaume na wanawake Tanzania',
+        'author': 'Nuzulack Dausen',
+        'brief': 'Dar es Salaam. Licha ya kipato cha mfanyakazi kwa mwezi kuongezeka'\
+        ' nchini bado kuna mwanya mkubwa wa kimapato kijinsia baada ya wanaume kuingiza fedha nyingi zaidi kuliko wanawake.',
+        'link': 'https://data.mwananchi.co.tz/hii-ndiyo-siri-ya-utofauti-wa-kipato-kati-ya-wanaume-na-wanawake-tanzania/',
+        'img': STATIC_URL + 'img/showcase/kipato.png'
+     },
+    {
+        'title': 'CTI calls streamlined taxation food processing sector',
+        'author': 'IPP Media',
+        'brief': 'A recent Confederation of Tanzania Industries (CTI) position'\
+        ' paper on regulatory authorities shows that although the manufacturing'\
+        ' sector generates tax revenues to the tune of 248 billion/- per year.',
+        'link': 'http://ippmedia.com/en/news/cti-calls-streamlined-taxation-food-processing-sector',
+        'img': STATIC_URL + 'img/showcase/IPPMedia.png'
+     }
+]
+
+STATIC_ROOT = 'hurumap_tz/static/'
```

Comment:
Did this need to change?

Reply:
I added the setting while troubleshooting CSS errors the `python manage.py collectstatic` command was having trouble  finding `STATIC_ROOT` folder.